src/components/CreateProject/CreateProjectModal.cy.tsx

```diff
@@ -0,0 +1,68 @@
+/// <reference types="cypress" />
+import { CreateProjectModal } from './CreateProjectModal';
+
+const requiredFieldMessage = 'این فیلد ضروری است';
+const minLengthMessage = 'نام پروژه باید حداقل ۳ حرف باشد';
+const projectNameSelector = '[data-test="project-name"]';
+const submitButtonSelector = '[data-test="submit-button"]';
+
+describe('Create Project Form', () => {
+  beforeEach(() => {
+    cy.mount(<CreateProjectModal opened onClose={() => undefined} />);
```

Comment:
```suggestion
    cy.mount(<CreateProjectModal opened onClose={noop} />);
```
silly but why not when we have a function for it :)
you can import it from `@fullstakcsjs/toolbox` (it's installed)

Reply:
tnx, it's reasonable